src/less/chat.less

```diff
@@ -136,8 +143,19 @@
     color: @sfrpgRed;
     font-weight: bold;
   }
+  .message-metadata {
```

Comment:
This is redundant. Foundry core `css` already defines these, so they aren't doing anything other than masking other styling that's already there.

Reply:
I think these were leftover from when I was trying to figure out why the `.whisper` and `.blind` text wasn't appearing on the GM's view.